platform/darwin/scripts/generate-style-code.js

```diff
@@ -157,26 +136,27 @@ global.parseColor = function (str) {
 global.describeValue = function (value, property, layerType) {
     switch (property.type) {
         case 'boolean':
-            return value ? '`YES`' : '`NO`';
+            return 'an `NSNumber` object containing ' + (value ? '`YES`' : '`NO`');
         case 'number':
+            return 'an `NSNumber` object containing the float `' + value + '`';
         case 'string':
-            return '`' + value + '`';
+            return 'the string `' + value + '`';
         case 'enum':
             let objCType = `${prefix}${camelize(layerType)}${suffix}${camelize(property.name)}`;
-            return '`' + `${objCType}${camelize(value)}` + '`';
+            return 'an `NSValue` object containing `' + `${objCType}${camelize(value)}` + '`';
         case 'color':
             let color = parseColor(value);
             if (!color) {
                 throw new Error(`unrecognized color format in default value of ${property.name}`);
             }
             if (color.r === 0 && color.g === 0 && color.b === 0 && color.a === 0) {
-                return '`clearColor`';
+                return '`NSColor.clearColor` or `UIColor.clearColor`';
```

Comment:
Rest assured, I was being completely objective and dispassionate by alphabetizing the items in this list. 😉